src/ui/PackageDependencyProvider.ts

```diff
@@ -176,16 +176,14 @@ export class PackageDependenciesProvider implements vscode.TreeDataProvider<Tree
                 first.name.localeCompare(second.name)
             );
         }
+
+        const buildDirectory = buildDirectoryFromWorkspacePath(folderContext.folder.fsPath);
```

Comment:
Build directory can't be relative here.

Reply:
Basically anything that uses vscode-uri can not use relative path. This is tricky for future support, imo. Worth discussing how we could make this safer in future